docs/api-docs/building-checkouts/checkout-customizability.md

```diff
@@ -17,9 +17,9 @@ Once a developer has chosen a path, they can choose the options they want to use
-| **Storefront Checkout API** | REST APIs to create and update a checkout and convert it to an order. <br/><br/>  | This option allows developers to build a custom checkout experience (within the context of a BigCommerce storefront session) by accessing raw API request/response data.<br/><br/> These are the APIs you would typically access from your browser using JavaScript to retrieve checkout data not available through the SDK. | [Storefront Checkout API](/api-reference/storefront/checkouts)<br/><br/> [Storefront Cart and Checkout](/api-docs/storefront/overview)|
-| **Server to server Checkout API** | REST APIs to create and update a checkout and convert it to an order. | This option allows developers to build a custom checkout (outside of a BigCommerce storefront session) and use the [Payments API](/api-reference/store-management/payment-processing) to pay for an order.<br/><br/> When used with the [Server to Server Cart API](/api-reference/store-management/carts) it allows you to make modifications similar to those performed using the control panel option **Orders > Add**. These modifications include the ability to create custom products, modify prices, add custom discounts, set custom prices for shipping, etc.<br/><br/> These are the APIs that will power your headless checkout solutions on channels such as: <ul><li> Content management systems</li> <li> Native mobile apps (non-web view)</li> <li> Internet of Things (IoT) devices</li> <li> Recurring order and/or subscription apps (e.g. OrderGroove)</li></ul> <br/> **Note:** Currently, this API currently does not support using store credit or gift certificates as payment methods. | [Server to Server Checkout API](/api-reference/store-management/checkouts) |
+|**Checkout JS SDK**| A client-side vanilla JavaScript library that is a wrapper for the Storefront Checkout API.<br/><br/> The SDK handles all the API calls necessary to move shoppers through the checkout experience. From customer login, fetching shipping quotes, and payment methods through to paying an order.| This option allows developers to build a custom checkout from scratch by creating a presentation layer for checkout. Completely replacing the UI/UX without having to do the heavy lifting of dealing with Storefront API endpoints.<br/><br/> Significant front-end customizations and/or feature additions to checkout (hosted on the BigCommerce storefront).<br/><br/> **Note:** SDK does not let you introduce new behavior. While you can filter based on shipping methods, you cannot create new shipping methods or payment methods. | [Checkout SDK](/api-docs/cart-and-checkout/checkout-sdk |
```

Comment:
```suggestion
|**Checkout JS SDK**| A client-side vanilla JavaScript library that is a wrapper for the Storefront Checkout API.<br/><br/> The SDK handles all the API calls necessary to move shoppers through the checkout experience. From customer login, fetching shipping quotes, and payment methods through to paying an order.| This option allows developers to build a custom checkout from scratch by creating a presentation layer for checkout. Completely replacing the UI/UX without having to do the heavy lifting of dealing with Storefront API endpoints.<br/><br/> Significant front-end customizations and/or feature additions to checkout (hosted on the BigCommerce storefront).<br/><br/> **Note:** SDK does not let you introduce new behavior. While you can filter based on shipping methods, you cannot create new shipping methods or payment methods. | [Checkout SDK](/stencil-docs/customizing-checkout/checkout-sdk) |
```

Reply:
redirects indicate that this is the preferred human-readable link. also md close bracket.